dbot/dbot/CommandModules/ViewModule.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Discord;
+using Discord.Commands;
+using Discord.WebSocket;
+using dbot.Services;
+using System.Threading.Tasks;
+
+namespace dbot.CommandModules
+{
+    [Group("view")]
+    public class ViewModule : ModuleBase
+    {
+        private readonly NominationsService _nominationsService;
+        private readonly OmdbService _omdbService;
+        private readonly VotingService _votingService;
+
+        public ViewModule(NominationsService ns, OmdbService os, VotingService vs)
+        {
+            _nominationsService = ns;
+            _omdbService = os;
+            _votingService = vs;
+        }
+
+        [Command("nominations")]
+        public async Task viewNom() {
+            var noms = _nominationsService.getNominations();
+            if (noms.Equals(null)) { return; }
```

Comment:
if(!noms.Any()) would be better, if nominations service can return null I think it should just return an IEnumerable<NomObj>.Empty instead

Reply:
Generally linq methods with no results will return empties to enable method chaining so it probably already is doing that